src/parser/sparqlParser/generated/SparqlAutomaticBaseListener.cpp

```diff
@@ -1,4 +1,4 @@
 
-// Generated from SparqlAutomatic.g4 by ANTLR 4.11.1
+// Generated from SparqlAutomatic.g4 by ANTLR 4.13.2
```

Comment:
If we now default to a newer version, we should also adapt the `Readme.md` in the `generated` folder.